examples/stringification/main.cpp

```diff
@@ -0,0 +1,491 @@
+#else
+    add_stub_asserts(6);
+#endif
+}
+
+TEST_CASE("Enumerations")
+{
+    CHECK_EQ(toString(Enum::E_VALUE), "1");
+    CHECK_EQ(toString(EnumClass::VALUE), "3");
+#ifdef ENABLE_BROKEN_TESTS
+    CHECK_EQ(toString(EnumWithOss::E_VALUE_WITH_OSS), "E_VALUE_WITH_OSS");
+    CHECK_EQ(toString(EnumClassWithOss::A), "A");
+#else
+    add_stub_asserts(2);
+#endif
+}
+
+TEST_CASE("Structs")
+{
+    CHECK_EQ(toString(Struct{}), "{?}"); // No overloaded operator, no text representation
+    CHECK_EQ(toString(StructOss{7}), "7"); // Use overloaded ostream operator
+}
+
+TEST_CASE("std_classes")
+{
+#if CXX_STANDARD >= 11
+    
+    {
+        const std::string value = "Hello world!";
+        CHECK_EQ(toString(value), "Hello world!");
+    }
+
+    {
+        std::pair<int, float> value { 1, 2.3f };
+        CHECK_EQ(toString(value), "{?}");
+    }
+
+    {
+        std::tuple<bool, int, double> value { false, 7, 6.6 };
+        CHECK_EQ(toString(value), "{?}");
+    }
+    
+    {
+        std::array<int, 3> value {{ 1, 5, 8 }};
+        CHECK_EQ(toString(value), "{?}");
+    }
+
+    {
+        std::valarray<int> value { 7, 3, 2 };
+        CHECK_EQ(toString(value), "{?}");
+    }
+
+    {
+        std::initializer_list<int> value { 1, 7, 0 };
+        CHECK_EQ(toString(value), "{?}");
+    }
+
+    {
+        std::vector<int> value { 4, 2, 7 };
+        CHECK_EQ(toString(value), "{?}");
+    }
+
+    {
+        std::deque<int> value { 1, 1, 6, 2 };
+        CHECK_EQ(toString(value), "{?}");
+    }
+
+    {
+        std::list<int> value { 7, 5, 2, 1 };
+        CHECK_EQ(toString(value), "{?}");
+    }
+
+    {
+        std::forward_list<int> value { 7, 0, 3, 7, 4 };
+        CHECK_EQ(toString(value), "{?}");
+    }
+
+    {
+        std::stack<int> value {{ 1, 2, 3, 4, 5 }};
+        CHECK_EQ(toString(value), "{?}");
+    }
+
+    {
+        std::queue<int> value {{ 1, 2, 3, 4, 5 }};
+        CHECK_EQ(toString(value), "{?}");
+    }
+
+    {
+        const auto data = { 1, 8, 5, 6, 3 };
+
+        std::priority_queue<int> value { data.begin(), data.end() };
+        CHECK_EQ(toString(value), "{?}");
+    }
+    
+    {
+        std::set<int> value {{ 1, 2, 2, 6, 1, 4 }};
+        CHECK_EQ(toString(value), "{?}");
+    }
+    
+    {
+        std::multiset<int> value {{ 1, 2, 2, 6, 1, 4 }};
+        CHECK_EQ(toString(value), "{?}");
+    }
+    
+    {
+        std::map<int, std::string> value {{ { 1, "foo" }, { 2, "bar" }, { 1, "baz" } }};
+        CHECK_EQ(toString(value), "{?}");
+    }
+    
+    {
+        std::multimap<int, std::string> value {{ { 1, "foo" }, { 2, "bar" }, { 1, "baz" } }};
+        CHECK_EQ(toString(value), "{?}");
+    }
+#else
+    add_stub_asserts(17);
+#endif  // CXX_STANDARD >= 11
+
+#if CXX_STANDARD >= 14
+    {
+        std::integer_sequence<int, 9, 2, 5, 1, 9, 1, 6> value {};
+        CHECK_EQ(toString(value), "{?}");
+    }
+#else
+    add_stub_asserts(1);
```

Comment:
Both cases are invalid. I've just tried them both.

-  "-minimal" completely removes output in console output format, but it doesn't affect XML and junit output formats. They still contain assert statistics
- NO_OUTPUT is either broken or intended for compile-time checks only. Because the output is ignored and not compared, retcode is completely ignored. So any test no matter of failed asserts passed as a result. In the initial version, I added alternative function doctest_add_test_2 just to workaround this behaviour